tempodb/encoding/vparquet2/combiner.go

```diff
@@ -160,6 +160,7 @@ func (c *Combiner) Result() (*Trace, int) {
 	if c.result != nil && c.combined {
 		// Only if anything combined
 		SortTrace(c.result)
```

Comment:
idle thought: since we are already sorting the trace can we dedupe batches here to reduce parquet size and improve performance on resource level attributes? wdyt?

Reply:
I think that sounds like something that's worth to look into. But I guess that's probably better handled in a separate issue. Should I create one?